components/collector/src/source_collectors/jira/lead_time_for_changes.py

```diff
@@ -0,0 +1,42 @@
+"""Jira lead time for changes collector."""
+
+from statistics import mean
+from typing import cast
+
+from dateutil.parser import parse
+
+from collector_utilities.functions import days_ago
+from collector_utilities.type import Value, URL
+from model import Entities, Entity
+
+from .issues import JiraIssues
+
+
+class JiraLeadTimeForChanges(JiraIssues):
+    """Jira collector for lead time for changes."""
+
+    def _include_issue(self, issue: dict) -> bool:
+        """Return whether this issue should be counted."""
+        if issue["fields"]["status"]["statusCategory"]["key"] != "done":
+            return False
+        if not (finished_date := issue["fields"].get("updated")):
+            return False
+        return days_ago(parse(finished_date)) <= int(cast(str, self._parameter("lookback_days")))
+
+    def _create_entity(self, issue: dict, url: URL) -> Entity:
+        """Extend to also add the lead time to the entity."""
+        entity = super()._create_entity(issue, url)
+        entity["lead_time"] = self.__lead_time(issue["fields"])
+        return entity
+
+    def _compute_value(self, entities: Entities) -> Value:
+        """Calculate the average lead time of the completed issues."""
+        lead_times = []
+        for issue in entities:
+            lead_times.append(self.__lead_time(issue))
+        return str(round(mean(lead_times))) if entities else None
```

Comment:
I don't understand why you changed `_compute_value` from a class method to an instance method? It doesn't use any instance attributes?

Reply:
Hmm, I probably auto-refactored all the other ones when creating this method as an instance method.
Does it make sense to also refactor the other instance methods to classmethods, as we almost never use instance attributes?

Reply:
No, that's ok. It's not a big deal, but I was just wondering what I missed :-)